src/pages/signin/LoginForm.js

```diff
@@ -77,7 +81,7 @@ class LoginForm extends React.Component {
                         autoCapitalize="none"
                         placeholder="Phone or Email"
                         placeholderTextColor={themeColors.placeholderText}
-                        autofocus
+                        autofocus={!this.props.isSmallScreenWidth}
```

Comment:
you missed what I was trying to hint about 😄 

re-read my comment, test on all platforms, and maybe try again?

Reply:
For iOS devices we default to the narrow view so there's no chance the keyboard will get pulled up and cover and promo material.

But at the same time I tried it on web and it didn't autofocus 🤔 I'll just get rid of this entirely for now

Reply:
>implying that autofocus just doesn't even have an effect.

Were you able to figure out why it has no effect?

>I'll just get rid of this entirely for now.

To be clear, I think we maybe _do_ want this to focus when:

1. Everywhere, when we are on web/desktop (excluding mobile web)
2. All screens except the very first screen on mobileWeb/native

Reply:
@marcaaron Currently the only way to have autofocus disabled for wide views on native is using an `index.native` file which is exactly what this PR is trying to avoid (having multiple components). Unless we write a HOC that checks whether we're on native or on web which from my research is kind of hacky (https://stackoverflow.com/questions/31732342/is-there-a-way-in-react-to-tell-if-it-is-react-native-or-web). What are your thoughts?